internal/backend/runtime/helpers/jsonschema.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2025 Sidero Labs, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+
+package helpers
```

Comment:
Don't want to nitpick much, but since we also want to organize things a bit better, what do you think about creating a `jsonschema` package and moving these into it? The function names would then be `Parse` and `Validate`.

Reply:
makes sense, let's do it